packages/x-randomvalues/src/browser.ts

```diff
@@ -1,8 +1,13 @@
 // Copyright 2017-2022 @polkadot/x-randomvalues authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import { xglobal } from '@polkadot/x-global';
+
 export { packageInfo } from './packageInfo';
 
```

Comment:
It seems works for Browser and Deno.
But, can we just add this line instead?

```suggestion
  let crypto = xglobal.crypto || require('crypto').webcrypto;
```

It will also support for Node.js if the developer prefer to import the bundled version. But I think it's low priority for now, and it may only happen if Node.js users is importing through [HTTPS module loader](https://nodejs.org/api/esm.html#https-loader) instead of installing through package manager.

Reply:
The bundled is not for node. If we wish to support node, it would mean adding an `x-crypto` and using export maps - we don't use inline requires anywhere.

Reply:
Hmm, ok, let's forget about it as using package manager is already a working solution for Node.js.